CHANGELOG.md

```diff
@@ -55,6 +56,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Improve summary error message for invalid setting updates ([#4792](https://github.com/opensearch-project/OpenSearch/pull/4792))
 - Return 409 Conflict HTTP status instead of 503 on failure to concurrently execute snapshots ([#8986](https://github.com/opensearch-project/OpenSearch/pull/5855))
 - Performance improvement for Datetime field caching ([#4558](https://github.com/opensearch-project/OpenSearch/issues/4558))
+- Made leader/follower check timeout setting dynamic ([#10528](https://github.com/opensearch-project/OpenSearch/pull/10528))
```

Comment:
@niyatiagg no need to add entry here. Since this change will be backported to 2.x. So, just one entry below would work. Also, can you please resolve the conflicts?

Reply:
> Also, can you please resolve the conflicts?

Done

Reply:
Not fully resolved.
<img width="1580" alt="Screenshot 2023-10-16 at 5 32 53 PM" src="https://github.com/opensearch-project/OpenSearch/assets/16099877/2c23b9fd-7564-42b2-9c8a-e085abb8c9d8">

Reply:
Sorry, my bad! Should be sorted now.